frontend/tailwind.config.js

```diff
@@ -130,22 +108,15 @@ module.exports = {
       ],
     },
     fontSize: {
-      '13': '13px',
-      '14': '14px',
-      '15': '15px',
-      '16': '16px',
-      '17': '17px',
-      '18': '18px',
-      '20': '20px',
-      '22': '22px',
-      '24': '24px',
-      '30': '30px',
-      '35': '35px',
-      '40': '40px',
-      '50': '50px',
-      '62': '62px',
-      '66': '66px',
-      '70': '70px',
+      'base': '10px', 
```

Comment:
Set the mobile base to 10px to make calculations easy, but this could be anything we want.

Reply:
Do you mean rem calculations?

Reply:
Yes. If html has font-size 10px then 2rem would be 20px, 3rem would be 30px, etc.